python/ray/tune/trial.py

```diff
@@ -301,7 +323,7 @@ def __init__(self,
             if self._cmp_greater else checkpoint_score_attr[4:]
 
         self._checkpoint = Checkpoint(
-            storage=Checkpoint.DISK, value=restore_path)
+            storage=Checkpoint.DISK, value=_find_newest_ckpt(restore_path))
```

Comment:
Can you add the documentation for this in `tune.py` under the `restore` arg?

Reply:
> Can you add the documentation for this in `tune.py` under the `restore` arg?

Done, thanks!